linux/LinuxMDEparser/README.ME

```diff
@@ -0,0 +1,23 @@
+# LinuxMDEparser
+Microsoft Defender Endpoint Log Parser for Linux and MacOS.
+
+Compiled sources (with pyinstaller) can be found on dist folder.
+
+# About
+The LinuxMDEparser is a tool to export multiple log files from MDE for Linux and Mac to Excel, making it easier to filter and search the logs.
+
+# Help
+You can always check the available options by running: LinuxMDEparser.exe -h
```

Comment:
It is a .py

Reply:
Corrected

Reply:
It still shows as `LinuxMDEparser.exe` to me